plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTest.java

```diff
@@ -86,8 +91,6 @@ public void getPluginSignals() {
 
     SignalInfo assetPackStateUpdateSignal =
         new SignalInfo("assetPackStateUpdated", Dictionary.class);
-    SignalInfo fetchStateUpdated =
-        new SignalInfo("fetchStateUpdated", Dictionary.class, Integer.class);
     SignalInfo fetchSuccess = new SignalInfo("fetchSuccess", Dictionary.class, Integer.class);
```

Comment:
I decided not do use const String variables here since these tests help us to ensure if there are any typos in signals names.